app/src/main/java/org/mozilla/fenix/settings/wallpaper/WallpaperSettings.kt

```diff
@@ -269,33 +269,35 @@ private fun WallpaperThumbnailItem(
 }
 
 @Composable
+@Suppress("MagicNumber")
 private fun WallpaperLogoSwitch(
     checked: Boolean,
     onCheckedChange: (Boolean) -> Unit
 ) {
-    Column(
-        modifier = Modifier.padding(horizontal = 12.dp, vertical = 16.dp),
+    Row(
+        horizontalArrangement = Arrangement.SpaceBetween,
```

Comment:
Can we move this below `modifier` just so we are following the parameter ordering.